.github/workflows/browserstack.yaml

```diff
@@ -36,4 +36,4 @@ jobs:
         env:
           BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
           BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
-        run: npx turbo run test-browserstack --output-logs new-only --log-order stream
+        run: npx turbo run test-browserstack
```

Comment:
We should keep the flags. For long-running tasks such as this, it can be very useful to have the unbuffered logs.